flink-core/src/main/java/org/apache/flink/configuration/AkkaOptions.java

```diff
@@ -178,13 +170,12 @@ public static boolean isForceRpcInvocationSerializationEnabled(Configuration con
                     .defaultValue(true)
                     .withDescription("Exit JVM on fatal Akka errors.");
 
-    /** Milliseconds a gate should be closed for after a remote connection was disconnected. */
-    public static final ConfigOption<Long> RETRY_GATE_CLOSED_FOR =
-            ConfigOptions.key("akka.retry-gate-closed-for")
-                    .longType()
-                    .defaultValue(50L)
-                    .withDescription(
-                            "Milliseconds a gate should be closed for after a remote connection was disconnected.");
+    /** Retry outbound connection only after this backoff. */
+    public static final ConfigOption<String> OUTBOUND_RESTART_BACKOFF =
```

Comment:
I did this change based on the configuration comments.

classic doc:
```yaml
      # After failed to establish an outbound connection, the remoting will mark the
      # address as failed. This configuration option controls how much time should
      # be elapsed before reattempting a new connection. While the address is
      # gated, all messages sent to the address are delivered to dead-letters.
      # Since this setting limits the rate of reconnects setting it to a
      # very short interval (i.e. less than a second) may result in a storm of
      # reconnect attempts.
      retry-gate-closed-for = 5 s
```

artery doc:
```yaml
        # Retry outbound connection after this backoff.
        # Only used when transport is tcp or tls-tcp.
        outbound-restart-backoff = 1 second
```

The `outbound-restart-backoff` comments are a lot less specific, but they point to the same direction.

Reply:
> they point to the same direction

That was also my conclusion from the docs. Shall we add retry-gate-closed-for as a deprecated key to the new option?

Reply:
Yeah, that makes sense. I also thought about using the value of `retry-gate-closed-for` if that is set explicitly and `outbound-restart-backoff` is missing.